docs/api/v3.rst

```diff
@@ -143,29 +154,29 @@ Project details
                 "virtualenv"
             ],
             "users": [
-                "dstufft",
-                "pmoore",
-                "xafer",
-                "pradyunsg"
+                {
+                    "created": "2010-10-23T18:12:31+00:00",
```

Comment:
I don't think we can to expose this data do we?

Reply:
I'm removing it for now. We can add user's fields later if we consider without breaking anything.